On my M1 mac using `environment-accelerate` I am getting 2 failures in `test_FLEbasis2D.py`:

```
======================================================= FAILURES ========================================================
_______________________________________ TestFLEBasis2D.testFastVDense_T[33-1e-14] _______________________________________

self = <tests.test_FLEbasis2D.TestFLEBasis2D object at 0x10d9d27d0>
basis = <aspire.basis.fle_2d.FLEBasis2D object at 0x10d522680>

    def testFastVDense_T(self, basis):
    
        dense_b = basis._create_dense_matrix()
    
        # create sample particle
        x = create_images(basis.nres, 1).asnumpy()
        xvec = x.reshape((basis.nres**2, 1))
    
        # explicit matrix multiplication
        result_dense = dense_b.T @ xvec
        # fast evaluate_t
        result_fast = basis.evaluate_t(Image(x))
    
>       assert relerr(result_dense.T, result_fast) < (self.test_eps * basis.epsilon)
E       assert 1.3757207857828869e-14 < (1.3 * 1e-14)
E        +  where 1.3757207857828869e-14 = relerr(array([[ 1.74565159e-04,  2.64969817e-04, -5.73204746e-05,\n        -7.02058550e-05,  1.48038225e-04,  9.53922719e-06,\n...         3.96563485e-08,  1.49875335e-07,  6.23780917e-07,\n         1.60399891e-07,  4.02395306e-07, -1.71811276e-07]]), <aspire.basis.basis.Coef object at 0x10de46bf0>)
E        +    where array([[ 1.74565159e-04,  2.64969817e-04, -5.73204746e-05,\n        -7.02058550e-05,  1.48038225e-04,  9.53922719e-06,\n...         3.96563485e-08,  1.49875335e-07,  6.23780917e-07,\n         1.60399891e-07,  4.02395306e-07, -1.71811276e-07]]) = array([[ 1.74565159e-04],\n       [ 2.64969817e-04],\n       [-5.73204746e-05],\n       [-7.02058550e-05],\n       [ 1.480...49875335e-07],\n       [ 6.23780917e-07],\n       [ 1.60399891e-07],\n       [ 4.02395306e-07],\n       [-1.71811276e-07]]).T
E        +  and   1.3 = <tests.test_FLEbasis2D.TestFLEBasis2D object at 0x10d9d27d0>.test_eps
E        +  and   1e-14 = <aspire.basis.fle_2d.FLEBasis2D object at 0x10d522680>.epsilon

tests/test_FLEbasis2D.py:91: AssertionError
------------------------------------------------- Captured stdout call --------------------------------------------------
2026-04-29 09:25:45,854 INFO [aspire.source.image] Creating Simulation with 1 images.
--------------------------------------------------- Captured log call ---------------------------------------------------
INFO     aspire.source.image:image.py:220 Creating Simulation with 1 images.
DEBUG    aspire.nufft:__init__.py:237 nufft passed real_type for signal, converting
DEBUG    aspire.image.xform:xform.py:516 Applying forward transformations in pipeline
DEBUG    aspire.image.xform:xform.py:519 All forward transformations applied
________________________________________ TestFLEBasis2D.testFastVDense[33-1e-14] ________________________________________

self = <tests.test_FLEbasis2D.TestFLEBasis2D object at 0x10d9d16c0>
basis = <aspire.basis.fle_2d.FLEBasis2D object at 0x10d522680>

    def testFastVDense(self, basis):
    
        dense_b = basis._create_dense_matrix()
    
        # get sample coefficients
        x = create_images(basis.nres, 1)
        # hold input test data constant (would depend on epsilon parameter)
        coefs = FLEBasis2D(
            basis.nres, epsilon=1e-4, dtype=np.float64, match_fb=False
        ).evaluate_t(x)
    
        result_dense = dense_b @ coefs.asnumpy().T
        result_fast = basis.evaluate(coefs).asnumpy()
    
>       assert relerr(result_dense, result_fast) < (self.test_eps * basis.epsilon)
E       assert 1.4232905634715797e-14 < (1.3 * 1e-14)
E        +  where 1.4232905634715797e-14 = relerr(array([[0.],\n       [0.],\n       [0.],\n       ...,\n       [0.],\n       [0.],\n       [0.]]), array([[[0., 0., 0., ..., 0., 0., 0.],\n        [0., 0., 0., ..., 0., 0., 0.],\n        [0., 0., 0., ..., 0., 0., 0.],\n ...\n        [0., 0., 0., ..., 0., 0., 0.],\n        [0., 0., 0., ..., 0., 0., 0.],\n        [0., 0., 0., ..., 0., 0., 0.]]]))
E        +  and   1.3 = <tests.test_FLEbasis2D.TestFLEBasis2D object at 0x10d9d16c0>.test_eps
E        +  and   1e-14 = <aspire.basis.fle_2d.FLEBasis2D object at 0x10d522680>.epsilon

tests/test_FLEbasis2D.py:107: AssertionError
```

--- a/.github/workflows/workflow.yml
+++ b/.github/workflows/workflow.yml
@@ -159,6 +159,13 @@ jobs:
         os: [ubuntu-latest, ubuntu-22.04, macOS-latest, macOS-14]
         backend: [default, openblas]
         python-version: ['3.10']
+        # Skip testing x86_64 openblas on M1 arch
+        # Users should be targetting arm64 installs on M1
+        exclude:
+          - os: macOS-latest
+            backend: openblas
+          - os: macOS-14
+            backend: openblas
         include:
           - os: ubuntu-latest
             backend: intel

--- a/environment-accelerate.yml
+++ b/environment-accelerate.yml
@@ -9,6 +9,7 @@ dependencies:
   - python=3.10
   - numpy=1.25.0
   - scipy=1.13.1
+  - finufft=2.5.0  # must be installed via conda if using (conda) optimized backends
   - scikit-learn
   - scikit-image
   - libblas=*=*accelerate

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -32,7 +32,8 @@ dependencies = [
     "confuse == 2.1.0",
     "cvxpy",
     "finufft==2.5.0 ; sys_platform!='darwin' or (sys_platform=='darwin' and platform_machine!='x86_64')",
-    "finufft==2.3.0 ; sys_platform=='darwin' and platform_machine=='x86_64'",
+    # wheel for 2.5.0 on x86_64 not available. x86_64 macs are phasing out.
+    "finufft==2.4.1 ; sys_platform=='darwin' and platform_machine=='x86_64'",
     "gemmi >= 0.6.5",
     "joblib",
     "matplotlib >= 3.2.0",

